justfile

```diff
@@ -37,12 +37,12 @@ test-superchain:
 	go test ./superchain/... -v
 
 # Test all Go code in the validation module
-test-validation:
-	go test ./validation/... -v
+test-validation: clean-add-chain
+	TEST_DIRECTORY=./validation go run gotest.tools/gotestsum@latest --format testname
 
 # Run validation checks for chains with a name or chain ID matching the supplied regex, example: just validate 10
 validate CHAIN:
```

Comment:
Since we changed the readme to advise using chainId, would be nice to sync this comment and arg:
```
validate CHAIN_ID
```

Reply:
[4602f05](https://github.com/ethereum-optimism/superchain-registry/pull/388/commits/4602f0594d098b660a0a0b07f160ad20ea0d0ec1)